sample/url-ping/handler.py

```diff
@@ -8,11 +8,12 @@ def print_url(url):
         print("Timed out trying to reach URL.")
 
 def handle(req):
-    print("Handle this -> " + req)
-    if req.find("http") == -1:
+    url = req.rstrip()
```

Comment:
This is a good idea. You can also invoke this with echo -n "https://www.openfaas.com".